contribs/gmf/src/datasource/ExternalDataSourcesManager.js

```diff
@@ -331,6 +333,17 @@ export class ExternalDatSourcesManager {
     const id = getId(layer);
     const service = capabilities.Service;
 
+    /**@type {string}*/
+    let language;
+
+    const paramsString = url.split('?')[1];
+    if (paramsString && paramsString.length > 1) {
+      const searchParams = new URLSearchParams(paramsString);
+      language = searchParams.get('lang');
+    } else {
+      language = getBrowserLanguage(['de', 'fr', 'it']);
+    }
```

Comment:
I think we have a standard way in ngeo to get the language, wight @fredj

Reply:
Apart from the mesure tools there's no other component using the language.
I think the best solution could be to inject the `StatemanagerService` into the `ExternalDataSourcesManager` and then get the lang from it.